src/main/java/frc/robot/Robot.java

```diff
@@ -10,7 +10,8 @@
 import edu.wpi.first.wpilibj.TimedRobot;
 import edu.wpi.first.wpilibj.smartdashboard.SendableChooser;
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
-
+import edu.wpi.first.wpilibj.GenericHID.Hand;
+import edu.wpi.first.wpilibj.Compressor; 
```

Comment:
Is this unused? For now, we might want to consider removing it unless we are going to do the work in this branch/PR to make it use of it.